### ISSUE TITLE
Description about the structure of the document

### ISSUE THREAD

**Issue by hegyhati** [hegyhati/Diploma_Work_Template#2]

**Comment by oliverosz** [comment]
Resolved by d008c379

**Comment by hegyhati** [comment]
Not really, this one is (or ought to be) more about the structure of the thesis itself. It is more-or-less done in the main tex file, but I still want to add a little bit of explanation about the style, supposed content, preferred length of each chapter.
